user_data/strategies/Strategy002.py

```diff
@@ -168,10 +168,8 @@ def confirm_trade_entry(self, pair: str, order_type: str, amount: float, rate: f
         mode = "test"
         coin = pair.split("/")[0]
         brain = "Freq_" + self.__class__.__name__
-        if IS_BACKTEST:
```

Comment:
need IS_BACK_TEST logic here

Reply:
I took care of that inside the utill

Reply:
As you suggested to make the code in strategy as clean as possible

Reply:
yes, if IS_BACKTEST -> backtest else launcher

Reply:
this is done in util.py as I mentioned so should I consider it as resolved?